vividus-tests/src/main/resources/story/system/mobile_app/ApplicationStepsTests.story

```diff
@@ -3,7 +3,21 @@ Description: Integration tests for ApplicationSteps class
 Meta:
     @epic vividus-plugin-mobile-app
 
+Lifecycle:
+Examples:
+/data/tables/system/mobile_app/locators/${target-platform}.table
+
 Scenario: Verify step: 'Given I start mobile application with capabilities:$capabilities'
+Meta:
+    @healthCheck
 Given I start mobile application with capabilities:
 |name|value     |
 |app |${app-url}|
+
+
+Scenario: Verify step: 'Then("number of $state elements found by `$locator` is $comparisonRule `$quantity`' and Appium XPath Locator
+Then number of elements found by `xpath(<mainViewHeaderTextXpath>)` is equal to `1`
+
+
+Scenario: Verify step: 'Then number of $state elements found by `$locator` is $comparisonRule `$quantity`'
+Then number of NOT_SELECTED elements found by `xpath(<mainViewHeaderTextXpath>)` is equal to `1`
```

Comment:
what is `not_selected` here?

Reply:
I just used this to cover the step, all elements besides checkboxes, selects are always not selected, changed to visible

Reply:
but it doesn't make sense, what about using `ENABLED`?

Reply:
discussed to use visible and visibility all